compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -520,7 +520,14 @@ object desugar {
           enumCases.last.pushAttachment(DesugarEnums.DefinesEnumLookupMethods, ())
         val enumCompanionRef = TermRefTree()
         val enumImport =
-          Import(enumCompanionRef, enumCases.flatMap(caseIds).map(ImportSelector(_)))
+          Import(enumCompanionRef, enumCases.flatMap(caseIds).map{
+            enumCase => 
+              val zeroSpan = if enumCase.span.exists then
```

Comment:
.endPos already checks .exists so I don't think the if is necessary here. Also is there any specific reason to use .endPos? Usually we use .focus to get a zero-span position.

Reply:
> Usually we use .focus to get a zero-span position.

Actually historically it looks like I used .startPos so we might as well keep doing that: https://github.com/lampepfl/dotty/pull/2532/commits/f24190d5c60c4e019fba97425b6f3b10167a039d

Reply:
Ach, I should have refreshed the comments. Changed now to `startPos`